src/components/pagination/page/page.jsx

```diff
@@ -12,24 +13,27 @@ class Page extends Component {
   };
 
   render() {
-    const { classes, isSelected, isFirst, isLast, labelText, children } = this.props;
+    const { classes, isSelected, isFirst, isLast, labelText, url, children } = this.props;
+    const disabled = isSelected;
+    const Element = url && !disabled ? Link : 'button';
+    const customProps = Element === Link ? { to: url } : { type: 'button', disabled };
+
     return (
       <li
         className={cn(classes.item, {
           [classes.itemAlwaysVisible]: isSelected || isFirst || isLast,
         })}
       >
-        <button
-          type="button"
+        <Element
+          {...customProps}
           className={cn(classes.button, {
             [classes.buttonSelected]: isSelected,
           })}
           onClick={this.onPageClick}
-          disabled={isSelected}
           aria-label={labelText}
         >
           {children}
-        </button>
+        </Element>
```

Comment:
what do you think of making internal Element to be controllable from the outside? like `renderItem` props which will get all relevant data from this component to be able relevant Element?

Reply:
Sounds like a good idea!

Reply:
@iamstarkov Not sure what you mean